automated-testing/unit-testing/why-unit-tests.md

```diff
@@ -0,0 +1,42 @@
+# Why Unit Tests
+
+It is no secret that writing unit tests is hard, and even harder to write well. Writing unit tests also increases the
+development time for every feature. So why should we bother writing them?
+
+## Reduce costs
+
+There is no question that the later a bug is found, the more expensive it is to fix; especially so if the bug makes it
+into production. A [2008 research study by IBM](ftp://ftp.software.ibm.com/software/rational/info/do-more/RAW14109USEN.pdf)
+estimates that a bug caught in production could cost 6 times as much as if it was caught during implementation.
+
+## Increase Developer Confidence
```

Comment:
I think this is more strongly stated as "Prove your code works (no matter what you change)". 

"Developer confidence" sounds like it's only for the developer. Really, test coverage is for developers, POs, architects, and anyone else with an investment in working code. For a DevOps engineer, it's the security that redeployment doesn't break anything. For POs, it's the confidence that their product (still) has all the capabilities they've paid for. For developers, it's the freedom to do any level of cleanup, refactoring, or rewrite without breaking anything. 

Also, "proving" is a key verb, here. Code with tests is provably working. Code without tests is a mystery.